evm_loader/cli/src/account_storage.rs

```diff
@@ -547,6 +553,18 @@ impl<'a> AccountStorage for EmulatorAccountStorage<'a> {
         self.block_timestamp.into()
     }
 
+    fn block_hash(&self, number: U256) -> H256 { 
+        info!("Get block hash {}", number);
+        let mut used_block_hash = self.used_block_hash.borrow_mut();
+        *used_block_hash = true;
+        if let Ok(timestamp) = self.config.rpc_client.get_block(number.as_u64()) {
+            H256::from_slice(&bs58::decode(timestamp.blockhash).into_vec().unwrap())
```

Comment:
Are you sure that `block_hash` in emulator will return the same value as in evm_loader program?
Can we have tests for this?

Reply:
Added test.